client/js/components/async-content.js

```diff
@@ -37,6 +39,15 @@ export default function asyncContent(el, dispatch) {
           containImages: false,
           scrollToClickedItem: false
         });
+
+        const transporterPromos = document.querySelectorAll('[data-track="transporter-promo"]');
+        nodeList(transporterPromos).forEach((transporterPromo) => {
+          const url = transporterPromo.href;
+
+          transporterPromo.addEventListener('click', () => {
+            trackEvent({name: 'Transporter promo', properties: {url}});
```

Comment:
Good point – would you take the URL out (i.e. is there any negative to it staying in? Drowning in data, maybe?).